src/main/kotlin/org/jetbrains/bio/experiments/fit/SpanModelFitExperiment.kt

```diff
@@ -453,4 +536,19 @@ abstract class SpanModelFitExperiment<out Model : ClassificationModel, State : A
             }
         }
     }
+}
+
+data class SpanDataPaths(
+        val treatment: Path,
+        val control: Path?
+)
+
+/**
+ * Legacy class for [SpanFitInformation] version 2.
+ */
+data class TC(
```

Comment:
Till the moment SPAN is not released and paper on monocytes ageing is not published, let's avoid any kind of legacy?

Reply:
You're probably right, I'll remove the version 2 compatibility.

Reply:
`SpanFitInfomation` is now versioned in the same way as `ClassificationModel`.